contrib/charts/templates/deployment-analyzer.yaml

```diff
@@ -56,7 +56,7 @@ spec:
         - name: SKYDIVE_UI_TOPOLOGY_FAVORITES
           value: '{"Infrastructure":"G.V().Has(\"Manager\", NE(\"k8s\"))","Kubernetes":"G.V().Has(\"Manager\", \"k8s\")"}'
         - name: SKYDIVE_UI_TOPOLOGY_DEFAULT_FILTER
-          value: 'G.V().Has("Manager", NE("k8s"))'
+          value: 'Infrastructure'
```

Comment:
Key has to be lower case, viper constraint

Reply:
Fixed and upload new commit move to lower case:

BTW: I checked and it worked with Uppercase ... so not sure about Viper 👍